### ISSUE TITLE
malformed jsmd should not leave the screen blank

### ISSUE THREAD

**Issue by hamilton** [iodide-project/iodide#561]
I tried checking a PR using a notebook made to test a new feature (freeze state) from another PR (which includes some new jsmd features), and when I loaded the notebook, I got the following:

<img width="794" alt="screen shot 2018-04-13 at 8 49 06 am" src="https://user-images.githubusercontent.com/95735/38744792-9f17ab3c-3ef7-11e8-9dc9-c1c7aea9ffab.png">

I would expect the behavior to be this:

- if jsmd is malformed (which it technically is on this branch), add an app error somewhere. For now having the console.error is a good first step
- load the editor and keep it blank

The msg is blocked by the snackbar, most likely, but we could probably unblock the "editor doesn't appear" part.

**Comment by bcolloran** [comment]
@hamilton can you  provide a reproducible test case?
